src/VirtualMonoRepo/allowed-binaries.txt

```diff
@@ -20,7 +20,7 @@
 **/TestCert*.pfx
 **/tests/*
 
-src/*/eng/common/loc/*.lss # UTF16-LE text files
+**/eng/common/loc/*.lss # UTF16-LE text files
```

Comment:
How about `src/**/eng/common/loc/*.lss`?

Reply:
That doesn't work either.

Reply:
I see